docs/development/setup-advanced.md

```diff
@@ -375,9 +410,14 @@ proxy in the environment as follows:
 
 ## Using Docker (experimental)
 
-Start by [cloning your fork of the zulip repository][zulip-rtd-git-cloning]:
+Start by [cloning your fork of the Zulip repository][zulip-rtd-git-cloning]
+and [connecting it to Zulip upstream][zulip-rtd-git-connect]:
```

Comment:
s/and connecting it to Zulip upstream/and adding the Zulip upstream repository as a remote/

(Or @gnprice do you have a better version of this to suggest?)

Reply:
Maybe "connecting the Zulip upstream repository"? Or the "adding ... as a remote" version is fine, just a little wordy.

Note the link is to a heading that reads "Connect your fork to Zulip upstream". That heading should match whatever we say here -- in particular it's wrong as it stands, because "your fork" means your repo on GitHub, and that's already connected to the Zulip upstream repo and isn't what you touch in that step.

Reply:
Unrelated tweak: the `git remote` help page tells me these instructions can be golfed a bit by inserting `-f` on `git remote add` and cutting out `git fetch upstream`.